src/components/AssetSearch/AssetList.tsx

```diff
@@ -27,23 +27,30 @@ export const AssetList: FC<AssetListProps> = ({
   hideZeroBalanceAmounts = true,
 }) => {
   const assetId = useRouteAssetId()
-  const [tokenListRef, setTokenListRef] = useRefCallback<FixedSizeList<AssetData>>({
+  const tokenListRef = useRef<FixedSizeList<AssetData> | null>(null)
+
+  useRefCallback<FixedSizeList<AssetData>>({
     deps: [assetId],
     onInit: node => {
       if (!node) return
-      const parsedAssetId = assetId ? decodeURIComponent(assetId) : undefined
-      const index = node.props.itemData?.assets.findIndex(
-        ({ assetId }: Asset) => assetId === parsedAssetId,
-      )
-      if (typeof index === 'number' && index >= 0) {
-        node.scrollToItem?.(index, 'center')
-      }
+      tokenListRef.current = node
     },
   })
 
   useEffect(() => {
-    if (!tokenListRef) return
-    tokenListRef?.scrollTo(0)
+    if (!tokenListRef.current) return
+    const parsedAssetId = assetId ? decodeURIComponent(assetId) : undefined
+    const index = tokenListRef.current.props.itemData?.assets.findIndex(
+      ({ assetId }: Asset) => assetId === parsedAssetId,
+    )
+    if (typeof index === 'number' && index >= 0) {
```

Comment:
Totally nitpick and potato potato, but can early return on eqeqequality against `-1` instead of using `typeof` if you feel like it (would have to also early return on `!tokenListRef.current.props.itemData` in that case)